assets/src/components/ui/styles/themes/default/index.js

```diff
@@ -1,6 +1,5 @@
 import './color-variables.css';
 import './size-variables.css';
-import '../../root/buttons.css';
```

Comment:
i can do that instead.
thought it would be better to only include the button styles if you were using the component that adds the classes targeted by the styles, ie: the `esprs-button` styles only apply to the `EspressoButton` component (unless you copy the setup and configuration of that component to another)